.changeset/modern-nails-approve.md

```diff
@@ -0,0 +1,16 @@
+---
+"@flopflip/react-broadcast": minor
+"@flopflip/react": patch
+---
+
+feat: add test context provider for react-broadcast
+
+You can wrap your application using for instance `react-testing-library` as
+
+```jsx
+import { FlopflipTestProvider } from '@flopflip/react-broadcast';
+
+<FlopflipTestProvider flags={testFlags} status={adapterStatus}>
+   <App />
+</FlopflipTestProvider>
```

Comment:
```suggestion
import { FlopflipTestProvider } from '@flopflip/react-broadcast';
import { render } from '@testing-library/react';

const testFlags = {
  myFlag: true,
};

describe('rendering', () => {
  it('should render the application with feature flags', () => {
    render(
      <FlopflipTestProvider flags={testFlags}>
        <App />
      </FlopflipTestProvider>
    );
  });
});
```

Reply:
Wow. Thanks!

Reply:
Well, since you mentioned `react-testing-library` you should provide an example for that.

You're welcome 😜